atomic_reactor/util.py

```diff
@@ -994,6 +994,26 @@ def get_field(self, field_name, args, kwargs):
         return (self.get_value(field_name, args, kwargs), field_name)
 
 
+class SessionWithTimeout(requests.Session):
+    """
+    requests Session with added timeout
+    """
+    def __init__(self, *args, **kwargs):
+        super(SessionWithTimeout, self).__init__(*args, **kwargs)
+
+    def get(self, *args, **kwargs):
```

Comment:
Wouldn't it be easier to override just `def request` there?

Reply:
not sure

Reply:
That's worth investigating.

Reply:
changed to override just request